onnxruntime/test/ir/onnx_model_test.cc

```diff
@@ -8,13 +8,22 @@
 #include "core/graph/model.h"
 #include "core/graph/op.h"
 #include "core/session/onnxruntime_c_api.h"
+#include "test/providers/provider_test_utils.h"  //For ASSERT_STATUS_OK
 #include "test/test_environment.h"
```

Comment:
Can we split the two macros that check the status into maybe test/util/include/asserts.h? It's a little weird to include a file from test/providers here.

Reply:
Right. We should do it.